session-replacement/src/main/java/com/amadeus/session/servlet/CookieSessionTracking.java

```diff
@@ -98,14 +98,17 @@ public void propagateSession(RequestWithSession request, Object response) {
   }
 
   private String cookiePath(HttpServletRequest request) {
-    return contextPath ? request.getContextPath() + "/" : "/";
+    if (contextPath != null && !contextPath.isEmpty()) {
+      return contextPath;
+    }
+    return request.getContextPath();
```

Comment:
If contextPath was not set, we should default to "/" as per https://docs.oracle.com/cd/E24329_01/web.1211/e21049/weblogic_xml.htm#WBAPP587